codecov.yml

```diff
@@ -1,6 +1,9 @@
 ---
 codecov:
   require_ci_to_pass: true
+  notify:
+    after_n_builds: 9
```

Comment:
Is this to stop the "This build decreases coverage by 0.000000000000007%" messages?

Reply:
YES!